tools/move-gemini.mjs

```diff
@@ -0,0 +1,139 @@
+/*!*
+ *
```

Comment:
Checking files top to bottom - sorry if resolved in further files and I forgot to remove, but: with no info and description this script has a high potential of not being used and getting dusty and obsolete pretty fast. For maintenance sake: could you add a description in the top comment?